paw-webapp/webapp/src/main/java/ar/edu/itba/paw/webapp/config/WebConfig.java

```diff
@@ -135,4 +137,9 @@ public JavaMailSender getMailSender() {
         mailSender.setJavaMailProperties(javaMailProperties);
         return mailSender;
     }
+
+    @Bean
+    public RestTemplate restTemplate() {
+        return new RestTemplate();
```

Comment:
Me parece medio overkill tener un Bean para algo que se usa en un solo lugar, sólo porque lo vamos a instanciar muchas veces (una por request si no es singleton). Tanto nos cuesta instanciar un `RestTemplate`?

Reply:
It's just best practice.

Igual para que comer recursos si se puede tener una única vez. Además, por que overkill? Es más overkill instanciar uno cada vez.

Reply:
Posta es best practice usar *siempre* un bean? Ya sé que te ahorrás instanciar el `RestTemplate` N veces así que está bueno el bean, pero si lo hubiese codeado yo lo hubiera instanciado en el constructor del `TwitchServiceImpl` y listo, total de eso creo que hay una sola instancia también. 
 Para mí el bean lo usás cuando hacés un recurso que es compartido por varias clases distintas, que no es el caso acá, por eso me pareció overkill.  Está perfecto lo que hiciste vos igual, era más opinión que otra cosa.

Reply:
Si lo instancias en el constructor, no estarías haciendo dependency injection.
Además, suponente que hacemos otra integración, se reusa.
Finalmente, tal vez se le quiera dar algún tipo de configuración al RestTemplate, y sería demasiada lógica de construcción del TwitchClient.